src/client/gui/lib/vm_table/bulk_actions.dart

```diff
@@ -68,6 +134,25 @@ class BulkActionsBar extends ConsumerWidget {
         ),
     ];
 
-    return Row(children: actionButtons.gap(width: 8).toList());
+    return Container(
+      height: 36,
+      margin: const EdgeInsets.only(top: 20),
+      child: Row(
+        children: [
+          Expanded(
+            child: SingleChildScrollView(
+              scrollDirection: Axis.horizontal,
+              child: Row(
+                children: [
+                  ...actionButtons.gap(width: 8).toList(),
+                  const SizedBox(width: 8),
+                  const ZonesDropdown(),
+                ],
+              ),
+            ),
+          ),
+        ],
+      ),
+    );
```

Comment:
Why was this necessary? It's a row that has a single expanded element inside it. And the items fit entirely even in a minimum size window, so the scroll view shouldn't be needed either?